test/uk/gov/hmrc/tai/connectors/RtiConnectorSpec.scala

```diff
@@ -38,7 +38,7 @@ import scala.concurrent.Future
 class RtiConnectorSpec extends ConnectorBaseSpec {
 
   val taxYear: TaxYear = TaxYear()
-  val url = s"/rti/individual/payments/nino/${nino.withoutSuffix}/tax-year/21-22"
+  val url = s"/rti/individual/payments/nino/${nino.withoutSuffix}/tax-year/${taxYear.twoDigitRange}"
```

Comment:
why does this need to change?

Reply:
because there's a dependency with TaxYear.now which picks up 22-23 since the 6th of April 😓